splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/CreateTableIT.java

```diff
@@ -224,4 +224,18 @@ public void testCreateTableIfNotExists_GlobalTempTable() throws Exception {
     public void testCreateTableIfNotExists_ExternalTable() throws Exception {
         testCreateTableIfNotExists("external");
     }
+
+    @Test
+    public void testCommentBeforeCreateTableAs() throws Exception {
+        methodWatcher.executeUpdate("create table test_comment_before_create_as_dep_tbl (col int)");
+
+        String createStmt = "create table %s as select * from test_comment_before_create_as_dep_tbl";
+        try {
+            methodWatcher.executeUpdate("-- some SQL-style comment here\n" + String.format(createStmt, generateTableName()));
+            methodWatcher.executeUpdate("/* some C-style comment here */\n" + String.format(createStmt, generateTableName()));
+            methodWatcher.executeUpdate("/* some mixed */\n-- comments\n" + String.format(createStmt, generateTableName()));
+        } catch (SQLException e) {
+            Assert.fail("No exception should be thrown");
```

Comment:
You don't need to catch anything here, just let the exception bubble up and if any is raised the test will fail with the appropriate stacktrace and message.

Reply:
Done.